### ISSUE TITLE
Bug: systemd service is all the time stopping due to the "Invalid ISO Extended String" error

### ISSUE THREAD

**Issue by slawqo** [abraunegg/onedrive#2964]
### Describe the bug

When I run `sudo systemd start onedrive@<username>` in journal log I see all the time errors like:

```
lis 12 10:06:40 p1gen4 systemd[1]: Starting onedrive@slaweq.service - OneDrive Client for Linux running for slaweq...
lis 12 10:06:55 p1gen4 systemd[1]: Started onedrive@slaweq.service - OneDrive Client for Linux running for slaweq.
lis 12 10:06:55 p1gen4 audit[1]: SERVICE_START pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='unit=onedrive@slaweq comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
lis 12 10:06:55 p1gen4 onedrive[4041778]: WARNING: Required environment variables required to enable GUI Notifications are not present
lis 12 10:06:55 p1gen4 onedrive[4041778]: WARNING: The required environment variables to enable GUI Notifications are not available; GUI notifications are disabled
lis 12 10:06:55 p1gen4 onedrive[4041778]: Using IPv4 and IPv6 (if configured) for all network operations
lis 12 10:06:55 p1gen4 onedrive[4041778]: Attempting to contact Microsoft OneDrive Login Service
lis 12 10:06:56 p1gen4 onedrive[4041778]: Successfully reached Microsoft OneDrive Login Service
lis 12 10:06:56 p1gen4 onedrive[4041778]: Configuring Global Azure AD Endpoints
lis 12 10:06:57 p1gen4 onedrive[4041778]: OneDrive synchronisation interval (seconds): 300
lis 12 10:06:57 p1gen4 onedrive[4041778]: Initialising filesystem inotify monitoring ...
lis 12 10:06:57 p1gen4 onedrive[4041778]: Performing initial synchronisation to ensure consistent local state ...
lis 12 10:06:57 p1gen4 onedrive[4041778]: Attempting to contact Microsoft OneDrive Login Service
lis 12 10:06:57 p1gen4 onedrive[4041778]: Successfully reached Microsoft OneDrive Login Service
lis 12 10:06:57 p1gen4 onedrive[4041778]: Starting a sync with Microsoft OneDrive
lis 12 10:06:58 p1gen4 onedrive[4041778]: Fetching items from the OneDrive API for Drive ID:  ..
lis 12 10:06:58 p1gen4 onedrive[4041778]: No changes or items that can be applied were discovered while processing the data received from Microsoft OneDrive
lis 12 10:06:59 p1gen4 onedrive[4041778]: Performing a database consistency and integrity check on locally stored data .Attempting to perform a database vacuum to optimise database
lis 12 10:06:59 p1gen4 onedrive[4041778]: Database vacuum is complete
lis 12 10:06:59 p1gen4 onedrive[4041778]: core.time.TimeException@/usr/lib/ldc/x86_64-redhat-linux-gnu/include/d/std/datetime/systime.d(9112): Invalid ISO Extended String: 2024-Nov-11 16:54:33.773
lis 12 10:06:59 p1gen4 onedrive[4041778]: ----------------
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x429ef4]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x426bbb]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x4268b2]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x456d7c]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x454ed9]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x456dc3]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x454ed9]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x456dc3]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x454ed9]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x4546da]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x4cf9ae]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x4ccf47]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? void rt.dmain2._d_run_main2(char[][], ulong, extern (C) int function(char[][])*).runAll() [0x7f5218cb134c]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? _d_run_main2 [0x7f5218cb1166]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? _d_run_main [0x7f5218cb0fbc]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x7f521852b247]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? __libc_start_main [0x7f521852b30a]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x4031c4]
lis 12 10:06:59 p1gen4 systemd[1]: onedrive@slaweq.service: Main process exited, code=exited, status=1/FAILURE
lis 12 10:06:59 p1gen4 systemd[1]: onedrive@slaweq.service: Failed with result 'exit-code'.
lis 12 10:06:59 p1gen4 audit[1]: SERVICE_STOP pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='unit=onedrive@slaweq comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=failed'
```

### Operating System Details

```shell
$ cat /etc/os-release                                                                                                                                                                    130 ↵
NAME="Fedora Linux"
VERSION="41 (KDE Plasma)"
RELEASE_TYPE=stable
ID=fedora
VERSION_ID=41
VERSION_CODENAME=""
PLATFORM_ID="platform:f41"
PRETTY_NAME="Fedora Linux 41 (KDE Plasma)"
ANSI_COLOR="0;38;2;60;110;180"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:41"
DEFAULT_HOSTNAME="fedora"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f41/system-administrators-guide/"
SUPPORT_URL="https://ask.fedoraproject.org/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=41
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=41
SUPPORT_END=2025-12-15
VARIANT="KDE Plasma"
VARIANT_ID=kde

$ uname -a
Linux p1gen4 6.11.5-300.fc41.x86_64 #1 SMP PREEMPT_DYNAMIC Tue Oct 22 20:11:15 UTC 2024 x86_64 GNU/Linux
```

### Client Installation Method

From Distribution Package

### OneDrive Account Type

Business | Office365

### What is your OneDrive Application Version

onedrive v2.5.2

### What is your OneDrive Application Configuration

```shell
onedrive --display-config
D-Bus message bus daemon is available; GUI notifications are now enabled
Application version                          = onedrive v2.5.2
Compiled with                                = LDC 2109
User Application Config path                 = /home/slaweq/.config/onedrive
System Application Config path               = /etc/onedrive
Applicable Application 'config' location     = /home/slaweq/.config/onedrive/config
Configuration file found in config location  = false - using application defaults
Applicable 'sync_list' location              = /home/slaweq/.config/onedrive/sync_list
Applicable 'items.sqlite3' location          = /home/slaweq/.config/onedrive/items.sqlite3
Config option 'drive_id'                     = 
Config option 'sync_dir'                     = ~/OneDrive
Config option 'enable_logging'               = false
Config option 'log_dir'                      = /var/log/onedrive
Config option 'disable_notifications'        = false
Config option 'skip_dir'                     = 
Config option 'skip_dir_strict_match'        = false
Config option 'skip_file'                    = ~*|.~*|*.tmp|*.swp|*.partial
Config option 'skip_dotfiles'                = false
Config option 'skip_symlinks'                = false
Config option 'monitor_interval'             = 300
Config option 'monitor_log_frequency'        = 12
Config option 'monitor_fullscan_frequency'   = 12
Config option 'read_only_auth_scope'         = false
Config option 'dry_run'                      = false
Config option 'upload_only'                  = false
Config option 'download_only'                = false
Config option 'local_first'                  = false
Config option 'check_nosync'                 = false
Config option 'check_nomount'                = false
Config option 'resync'                       = false
Config option 'resync_auth'                  = false
Config option 'cleanup_local_files'          = false
Config option 'classify_as_big_delete'       = 1000
Config option 'disable_upload_validation'    = false
Config option 'disable_download_validation'  = false
Config option 'bypass_data_preservation'     = false
Config option 'no_remote_delete'             = false
Config option 'remove_source_files'          = false
Config option 'sync_dir_permissions'         = 700
Config option 'sync_file_permissions'        = 600
Config option 'space_reservation'            = 52428800
Config option 'application_id'               = d50ca740-c83f-4d1b-b616-12c519384f0c
Config option 'azure_ad_endpoint'            = 
Config option 'azure_tenant_id'              = 
Config option 'user_agent'                   = ISV|abraunegg|OneDrive Client for Linux/v2.5.2
Config option 'force_http_11'                = false
Config option 'debug_https'                  = false
Config option 'rate_limit'                   = 0
Config option 'operation_timeout'            = 3600
Config option 'dns_timeout'                  = 60
Config option 'connect_timeout'              = 10
Config option 'data_timeout'                 = 60
Config option 'ip_protocol_version'          = 0
Config option 'threads'                      = 8
Environment var 'XDG_RUNTIME_DIR'            = true
Environment var 'DBUS_SESSION_BUS_ADDRESS'   = true
Config option 'notify_file_actions'          = false

Selective sync 'sync_list' configured        = false

Config option 'sync_business_shared_items'   = false

Config option 'webhook_enabled'              = false
```

### What is your 'curl' version

```shell
curl 8.9.1 (x86_64-redhat-linux-gnu) libcurl/8.9.1 OpenSSL/3.2.2 zlib/1.3.1.zlib-ng brotli/1.1.0 libidn2/2.3.7 libpsl/0.21.5 libssh/0.10.6/openssl/zlib nghttp2/1.62.1 OpenLDAP/2.6.8
Release-Date: 2024-07-31
Protocols: dict file ftp ftps gopher gophers http https imap imaps ipfs ipns ldap ldaps mqtt pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp ws wss
Features: alt-svc AsynchDNS brotli GSS-API HSTS HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM PSL SPNEGO SSL threadsafe TLS-SRP UnixSockets
```

### Where is your 'sync_dir' located

Local

### What are all your system 'mount points'

```shell
/dev/mapper/luks-133fc35a-1ab5-4b42-ab86-758c71aaa960 on / type btrfs (rw,relatime,seclabel,compress=zstd:1,ssd,discard=async,space_cache=v2,subvolid=257,subvol=/root)
devtmpfs on /dev type devtmpfs (rw,nosuid,seclabel,size=4096k,nr_inodes=8185648,mode=755,inode64)
tmpfs on /dev/shm type tmpfs (rw,nosuid,nodev,seclabel,inode64)
devpts on /dev/pts type devpts (rw,nosuid,noexec,relatime,seclabel,gid=5,mode=620,ptmxmode=000)
sysfs on /sys type sysfs (rw,nosuid,nodev,noexec,relatime,seclabel)
securityfs on /sys/kernel/security type securityfs (rw,nosuid,nodev,noexec,relatime)
cgroup2 on /sys/fs/cgroup type cgroup2 (rw,nosuid,nodev,noexec,relatime,seclabel,nsdelegate,memory_recursiveprot)
pstore on /sys/fs/pstore type pstore (rw,nosuid,nodev,noexec,relatime,seclabel)
efivarfs on /sys/firmware/efi/efivars type efivarfs (rw,nosuid,nodev,noexec,relatime)
bpf on /sys/fs/bpf type bpf (rw,nosuid,nodev,noexec,relatime,mode=700)
configfs on /sys/kernel/config type configfs (rw,nosuid,nodev,noexec,relatime)
proc on /proc type proc (rw,nosuid,nodev,noexec,relatime)
tmpfs on /run type tmpfs (rw,nosuid,nodev,seclabel,size=13106436k,nr_inodes=819200,mode=755,inode64)
tmpfs on /run/credentials/systemd-cryptsetup@luks\x2d133fc35a\x2d1ab5\x2d4b42\x2dab86\x2d758c71aaa960.service type tmpfs (ro,nosuid,nodev,noexec,relatime,nosymfollow,seclabel,size=1024k,nr_inodes=1024,mode=700,inode64,noswap)
selinuxfs on /sys/fs/selinux type selinuxfs (rw,nosuid,noexec,relatime)
systemd-1 on /proc/sys/fs/binfmt_misc type autofs (rw,relatime,fd=37,pgrp=1,timeout=0,minproto=5,maxproto=5,direct,pipe_ino=2425)
debugfs on /sys/kernel/debug type debugfs (rw,nosuid,nodev,noexec,relatime,seclabel)
mqueue on /dev/mqueue type mqueue (rw,nosuid,nodev,noexec,relatime,seclabel)
hugetlbfs on /dev/hugepages type hugetlbfs (rw,nosuid,nodev,relatime,seclabel,pagesize=2M)
tracefs on /sys/kernel/tracing type tracefs (rw,nosuid,nodev,noexec,relatime,seclabel)
tmpfs on /run/credentials/systemd-journald.service type tmpfs (ro,nosuid,nodev,noexec,relatime,nosymfollow,seclabel,size=1024k,nr_inodes=1024,mode=700,inode64,noswap)
tmpfs on /run/credentials/systemd-network-generator.service type tmpfs (ro,nosuid,nodev,noexec,relatime,nosymfollow,seclabel,size=1024k,nr_inodes=1024,mode=700,inode64,noswap)
tmpfs on /run/credentials/systemd-udev-load-credentials.service type tmpfs (ro,nosuid,nodev,noexec,relatime,nosymfollow,seclabel,size=1024k,nr_inodes=1024,mode=700,inode64,noswap)
fusectl on /sys/fs/fuse/connections type fusectl (rw,nosuid,nodev,noexec,relatime)
tmpfs on /run/credentials/systemd-sysctl.service type tmpfs (ro,nosuid,nodev,noexec,relatime,nosymfollow,seclabel,size=1024k,nr_inodes=1024,mode=700,inode64,noswap)
tmpfs on /run/credentials/systemd-tmpfiles-setup-dev-early.service type tmpfs (ro,nosuid,nodev,noexec,relatime,nosymfollow,seclabel,size=1024k,nr_inodes=1024,mode=700,inode64,noswap)
nfsd on /proc/fs/nfsd type nfsd (rw,relatime)
tmpfs on /run/credentials/systemd-tmpfiles-setup-dev.service type tmpfs (ro,nosuid,nodev,noexec,relatime,nosymfollow,seclabel,size=1024k,nr_inodes=1024,mode=700,inode64,noswap)
tmpfs on /run/credentials/systemd-cryptsetup@libvirt.service type tmpfs (ro,nosuid,nodev,noexec,relatime,nosymfollow,seclabel,size=1024k,nr_inodes=1024,mode=700,inode64,noswap)
tmpfs on /run/credentials/systemd-vconsole-setup.service type tmpfs (ro,nosuid,nodev,noexec,relatime,nosymfollow,seclabel,size=1024k,nr_inodes=1024,mode=700,inode64,noswap)
tmpfs on /tmp type tmpfs (rw,nosuid,nodev,seclabel,nr_inodes=1048576,inode64)
/dev/mapper/luks-133fc35a-1ab5-4b42-ab86-758c71aaa960 on /home type btrfs (rw,relatime,seclabel,compress=zstd:1,ssd,discard=async,space_cache=v2,subvolid=256,subvol=/home)
/dev/nvme0n1p2 on /boot type ext4 (rw,relatime,seclabel)
/dev/nvme0n1p1 on /boot/efi type vfat (rw,relatime,fmask=0077,dmask=0077,codepage=437,iocharset=ascii,shortname=winnt,errors=remount-ro)
/dev/mapper/libvirt on /var/lib/libvirt/images type ext4 (rw,relatime,seclabel,stripe=32)
binfmt_misc on /proc/sys/fs/binfmt_misc type binfmt_misc (rw,nosuid,nodev,noexec,relatime)
tmpfs on /run/credentials/systemd-tmpfiles-setup.service type tmpfs (ro,nosuid,nodev,noexec,relatime,nosymfollow,seclabel,size=1024k,nr_inodes=1024,mode=700,inode64,noswap)
sunrpc on /var/lib/nfs/rpc_pipefs type rpc_pipefs (rw,relatime)
tmpfs on /run/credentials/systemd-resolved.service type tmpfs (ro,nosuid,nodev,noexec,relatime,nosymfollow,seclabel,size=1024k,nr_inodes=1024,mode=700,inode64,noswap)
tmpfs on /run/user/1000 type tmpfs (rw,nosuid,nodev,relatime,seclabel,size=6553216k,nr_inodes=1638304,mode=700,uid=1000,gid=1000,inode64)
portal on /run/user/1000/doc type fuse.portal (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)
kio-fuse on /run/user/1000/kio-fuse-JJcQpN type fuse.kio-fuse (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)
```

### What are all your local file system partition types

```shell
lsblk -f
NAME                                          FSTYPE      FSVER LABEL                 UUID                                 FSAVAIL FSUSE% MOUNTPOINTS
zram0                                                                                                                                     [SWAP]
nvme0n1                                                                                                                                   
├─nvme0n1p1                                   vfat        FAT32                       FA38-C377                             579,4M     3% /boot/efi
├─nvme0n1p2                                   ext4        1.0                         7788814d-f2fd-4382-bda8-240b448f7431  588,4M    33% /boot
└─nvme0n1p3                                   crypto_LUKS 2                           133fc35a-1ab5-4b42-ab86-758c71aaa960                
  └─luks-133fc35a-1ab5-4b42-ab86-758c71aaa960 btrfs             fedora_localhost-live d8da56e2-7598-46dd-b7ee-80b1b2e2eda6  394,6G    58% /home
                                                                                                                                          /
nvme1n1                                                                                                                                   
└─nvme1n1p1                                   crypto_LUKS 2                           b02431b4-4c8d-43b2-9c55-121376fe6bd4                
  └─libvirt                                   ext4        1.0                         f0e057c7-7cb0-478b-9d7d-44729029ccf8  366,2G    55% /var/lib/libvirt/images
```

### How do you use 'onedrive'

I use same OneDrive on Windows laptop and on one Linux system. I sync one of my local directories on linux laptop to onedrive.

### Steps to reproduce the behaviour

run `systemctl start onedrive@<username>` and check journal log

### Complete Verbose Log Output

```shell
lis 12 10:06:40 p1gen4 systemd[1]: Starting onedrive@slaweq.service - OneDrive Client for Linux running for slaweq...
lis 12 10:06:55 p1gen4 systemd[1]: Started onedrive@slaweq.service - OneDrive Client for Linux running for slaweq.
lis 12 10:06:55 p1gen4 audit[1]: SERVICE_START pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='unit=onedrive@slaweq comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
lis 12 10:06:55 p1gen4 onedrive[4041778]: WARNING: Required environment variables required to enable GUI Notifications are not present
lis 12 10:06:55 p1gen4 onedrive[4041778]: WARNING: The required environment variables to enable GUI Notifications are not available; GUI notifications are disabled
lis 12 10:06:55 p1gen4 onedrive[4041778]: Using IPv4 and IPv6 (if configured) for all network operations
lis 12 10:06:55 p1gen4 onedrive[4041778]: Attempting to contact Microsoft OneDrive Login Service
lis 12 10:06:56 p1gen4 onedrive[4041778]: Successfully reached Microsoft OneDrive Login Service
lis 12 10:06:56 p1gen4 onedrive[4041778]: Configuring Global Azure AD Endpoints
lis 12 10:06:57 p1gen4 onedrive[4041778]: OneDrive synchronisation interval (seconds): 300
lis 12 10:06:57 p1gen4 onedrive[4041778]: Initialising filesystem inotify monitoring ...
lis 12 10:06:57 p1gen4 onedrive[4041778]: Performing initial synchronisation to ensure consistent local state ...
lis 12 10:06:57 p1gen4 onedrive[4041778]: Attempting to contact Microsoft OneDrive Login Service
lis 12 10:06:57 p1gen4 onedrive[4041778]: Successfully reached Microsoft OneDrive Login Service
lis 12 10:06:57 p1gen4 onedrive[4041778]: Starting a sync with Microsoft OneDrive
lis 12 10:06:58 p1gen4 onedrive[4041778]: Fetching items from the OneDrive API for Drive ID:  ..
lis 12 10:06:58 p1gen4 onedrive[4041778]: No changes or items that can be applied were discovered while processing the data received from Microsoft OneDrive
lis 12 10:06:59 p1gen4 onedrive[4041778]: Performing a database consistency and integrity check on locally stored data .Attempting to perform a database vacuum to optimise database
lis 12 10:06:59 p1gen4 onedrive[4041778]: Database vacuum is complete
lis 12 10:06:59 p1gen4 onedrive[4041778]: core.time.TimeException@/usr/lib/ldc/x86_64-redhat-linux-gnu/include/d/std/datetime/systime.d(9112): Invalid ISO Extended String: 2024-Nov-11 16:54:33.773
lis 12 10:06:59 p1gen4 onedrive[4041778]: ----------------
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x429ef4]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x426bbb]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x4268b2]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x456d7c]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x454ed9]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x456dc3]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x454ed9]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x456dc3]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x454ed9]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x4546da]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x4cf9ae]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x4ccf47]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? void rt.dmain2._d_run_main2(char[][], ulong, extern (C) int function(char[][])*).runAll() [0x7f5218cb134c]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? _d_run_main2 [0x7f5218cb1166]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? _d_run_main [0x7f5218cb0fbc]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x7f521852b247]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? __libc_start_main [0x7f521852b30a]
lis 12 10:06:59 p1gen4 onedrive[4041778]: ??:? [0x4031c4]
lis 12 10:06:59 p1gen4 systemd[1]: onedrive@slaweq.service: Main process exited, code=exited, status=1/FAILURE
lis 12 10:06:59 p1gen4 systemd[1]: onedrive@slaweq.service: Failed with result 'exit-code'.
lis 12 10:06:59 p1gen4 audit[1]: SERVICE_STOP pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='unit=onedrive@slaweq comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=failed'
```

**Comment by abraunegg** [comment]
@slawqo 
This particular issue has been fixed post v2.5.2 release via https://github.com/abraunegg/onedrive/pull/2882

Your options to resolve are:
* Build the client from source as per https://github.com/abraunegg/onedrive?tab=readme-ov-file#basic-troubleshooting-steps
* Wait for v2.5.3 release & packaging by your distribution

**Comment by abraunegg** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
